HW2_nekrasova/calculator.py

```diff
@@ -0,0 +1,62 @@
+def main(expression: str):
+    expression_array = expression.split(' ')
+    array_of_operands = []
+
+    try:
+        left_int = int(expression_array[0])
+        array_of_operands.append(left_int)
```

Comment:
```suggestion
        nums.append(left_int)
```

Логичнее было бы назвать переменную списком чисел, т.к. операнд - это +, -, /, *, и т.п.